src/Modules/CalcDefence.lua

```diff
@@ -289,14 +289,20 @@ function calcs.defence(env, actor)
 			end
 			armourBase = armourData.Armour or 0
 			if armourBase > 0 then
-				if slot == "Body Armour" and (modDB:Flag(nil, "Unbreakable") or modDB:Flag(nil, "DoubleBodyArmourDefence")) then
+				if slot == "Body Armour" and modDB:Flag(nil, "DoubleBodyArmourDefence") then
+					armourBase = armourBase * 2
+				end
+				if slot == "Body Armour" and modDB:Flag(nil, "Unbreakable") then
 					armourBase = armourBase * 2
```

Comment:
```suggestion
				if slot == "Body Armour" then
					if modDB:Flag(nil, "DoubleBodyArmourDefence") then
						armourBase = armourBase * 2
					end
					if modDB:Flag(nil, "Unbreakable") then
						armourBase = armourBase * 2
					end
```
You can get rid of one body armor comparison the same goes for the others it does increase the line count but I feel it is a little bit cleaner.

Reply:
I was also thinking that this would be a good idea

Reply:
That does indeed look better, good call!